armi/reactor/tests/test_components.py

```diff
@@ -49,6 +50,27 @@
     materials,
 )
 from armi.testing import loadTestReactor
+from armi.utils.units import getTc
+
+
+class CompositionDependentExpander(materials.Material):
```

Comment:
I apologize for myself.

But can we rename this something like `MockCompositionDependentExpander`?

I know that seems like a pedantic ask. But, no joke, we have had people import things from out tests classes for non-test uses in the past. And... #sigh. That if words like "mock" or "test" or "dummy" are included, I feel like we've done all we can.

Sorry.